src/webapp/src/auth/types.ts

```diff
@@ -1,16 +1,19 @@
-export interface User {
+export interface UserData {
   uid: string;
   email: string | null;
   displayName: string | null;
 }
+
 export enum AuthStatuses {
   notLoggedIn,
   loggedIn,
   loading,
 }
 
 export interface Login {
-  user: User | null;
+  uid: string | undefined;
+  email: string | undefined;
+  displayName: string | undefined;
   error?: string | null;
```

Comment:
Can we also have this as undefined?
```suggestion
  error?: string | undefined;
```

Reply:
Oh, I thought `null` was the correct value to use because we know we don't have an error at the beginning. I am going to ask you a little more about it later :ghost:

Reply:
Right! You right that makes sense